runelite-client/src/main/java/net/runelite/client/plugins/chatfilter/ChatFilterConfig.java

```diff
@@ -28,26 +28,25 @@
 import net.runelite.client.config.Config;
 import net.runelite.client.config.ConfigGroup;
 import net.runelite.client.config.ConfigItem;
+import net.runelite.client.config.ConfigSection;
 
 @ConfigGroup("chatfilter")
 public interface ChatFilterConfig extends Config
 {
-	@ConfigItem(
-		keyName = "filterType",
-		name = "Filter type",
-		description = "Configures how the messages are filtered",
-		position = 1
+	@ConfigSection(
+		name = "Filter Lists",
+		description = "Custom Word, Regex, and Username filter lists",
+		position = 0,
```

Comment:
Just make this position 1 like "Filter type" was before

Reply:
I was just following[ the precedent](https://github.com/runelite/runelite/commit/40f17648b4e83d1659d3fde5b7fb1d3ba2218a04#diff-637c5ee9b4bc1dae6fb2a818d2b34443) set by Hydrox, but I'm happy to change it if you feel I should.